src/gui/addnewtorrentdialog.h

```diff
@@ -38,6 +38,7 @@
 
 #include "base/bittorrent/infohash.h"
 #include "base/bittorrent/torrentinfo.h"
+#include "base/bittorrent/session.h"
```

Comment:
Not possible due to `AddNewTorrentDialog` having a member variable of type `BitTorrent::AddTorrentParams`.